tools/pbstate/src/pbstate.c

```diff
@@ -189,11 +192,10 @@ int pbstate_load(const char *p_device,
         backup_ok = false;
 
     if (!primary_ok && backup_ok) {
-        LOG("Primary configuration corrupt, using backup\n");
+        LOG("Primary state corrupt, using backup\n");
         memcpy(&config, &config_backup, sizeof(config));
     } else if (!primary_ok) {
-        LOG("Primary configuration corrupt\n");
-        LOG("Backup configuration corrupt\n");
+        LOG("Both primary and backup state corrupt\n");
         err = -1;
```

Comment:
Earlier in the method it returns an -errno, however here it is -1  (which maps ot EPERM, which is probably not what should be returned). Within a single method, either just have 0/-1, or 0/-errno, but don't mix

Reply:
fixed